### ISSUE TITLE
Update getting_started.md

### ISSUE THREAD

**Issue by b-k-schneider** [shorepine/alles#81]
HI, just trying out things from the documentation, and it seems that TARGET_DUTY, TARGET_FILTER_FREQ (and maybe others) are not working giving the error message (for example):
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'alles' has no attribute 'TARGET_DUTY'

Is this something to update in the documentation?
Commands found in: https://github.com/shorepine/alles/blob/main/getting-started.md

**Comment by bwhitman** [comment]
You're right -- we have not updated the getting started page. I know it's a bit messy, but please use the AMY README for more up to date documentation on the synth in Alles.  https://github.com/shorepine/amy

I will keep this open as a task to update this getting started page, thanks
